src/lang/proc_macros/controller.rs

```diff
@@ -203,6 +217,22 @@ impl ProcMacroClientController {
         }
     }
 
+    /// Sends `workspace/semanticTokens/refresh` if supported by the client to make sure macros
+    /// declared by proc macros are properly colored.
+    ///
+    /// Usage: should be called when the set of known macros is changed and all plugins with known
+    /// macros are already in the db.
```

Comment:
```suggestion
    /// Usage: should be called when the set of known macros is changed and all plugins with known
    /// macros are already in the db.
```
Can be called before as long as mutable `db` handler is not released